asset/css/markbind.css

```diff
@@ -40,3 +40,24 @@ blockquote {
         background-color: #fff;
     }
 }
+
+/* Footer */
+
+footer {
+    background-color: #f5f5f5;
+    color: dimgrey;
+    flex-shrink: 0;
```

Comment:
`flex-shrink: 0;` doesn't seem necessary, as discussed in https://github.com/MarkBind/markbind/pull/238#discussion_r191289525.